src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise.component.html

```diff
@@ -2,6 +2,9 @@
     <table class="table table-striped">
         <thead>
             <tr jhiSort [(predicate)]="predicate" [(ascending)]="reverse" (sortChange)="sortRows()">
+                <th *ngIf="course.isAtLeastEditor" class="d-md-table-cell">
```

Comment:
For programming exercises the editor check makes sense (since you can also bulk-edit here), but if only bulk-deleting is possible. this should only be shown for instructors.
```suggestion
                <th *ngIf="course.isAtLeastInstructor" class="d-md-table-cell">
```

Reply:
Thank you for your feedback, you are right. I changed it for the other exercises where only deletion is possible 👍 
Please have another look at the PR.